app/queries/find_by_property.rb

```diff
@@ -14,7 +14,7 @@ def initialize(query_service:)
 
   # Find by an arbitrary property. If property is :metadata, then value should
   # be a hash to query for the value of multiple properties.
-  def find_by_property(property:, value:, model: nil, lazy: false)
```

Comment:
It would be helpful to add to the comment about the created_at parameter, in particular its expected type and that it should be a range

Reply:
Done.